src/Components/CreateImageWizard/steps/Oscap/components/ProfileSelector.tsx

```diff
@@ -481,86 +350,52 @@ const ProfileSelector = () => {
-          onOpenChange={handleToggle}
-          toggle={toggleOpenSCAP}
-          shouldFocusFirstItemOnOpen={false}
-          popperProps={{
-            maxWidth: '50vw',
-          }}
-        >
-          <SelectList>
-            {isFetching && (
+    <FormGroup data-testid="profiles-form-group" label={<OpenSCAPFGLabel />}>
+      <Select
+        isScrollable
+        isOpen={isOpen}
+        selected={profileID}
+        onSelect={handleSelect}
+        onOpenChange={handleToggle}
+        toggle={toggleOpenSCAP}
+        shouldFocusFirstItemOnOpen={false}
+        popperProps={{
+          maxWidth: '50vw',
+        }}
+      >
+        <SelectList>
+          {isFetching && (
+            <SelectOption
+              value="loader"
+              data-testid="openscap-profiles-fetching"
+            >
+              <Spinner size="lg" />
+            </SelectOption>
+          )}
+          {selectOptions.length > 0 &&
+            [
               <SelectOption
-                value="loader"
-                data-testid="openscap-profiles-fetching"
+                key="oscap-none-option"
+                value={{ toString: () => 'None', compareTo: () => false }}
               >
-                <Spinner size="lg" />
-              </SelectOption>
-            )}
-            {selectOptions.length > 0 &&
-              [
-                <SelectOption
-                  key="oscap-none-option"
-                  value={{ toString: () => 'None', compareTo: () => false }}
-                >
-                  None
-                </SelectOption>,
-              ].concat(
-                selectOptions.map(
-                  (name: DistributionProfileItem, index: number) => (
-                    <OScapSelectOption key={index} profile_id={name} />
-                  )
+                None
+              </SelectOption>,
+            ].concat(
+              selectOptions.map(
+                (name: DistributionProfileItem, index: number) => (
+                  <OScapSelectOption key={index} profile_id={name} />
                 )
-              )}
-            {isSuccess && selectOptions.length === 0 && (
-              <SelectOption isDisabled>
-                {`No results found for "${filterValue}"`}
-              </SelectOption>
+              )
             )}
-          </SelectList>
-        </Select>
-      )}
-      {complianceType === 'compliance' && (
-        <Select
-          isScrollable
-          isOpen={isOpen}
-          onSelect={handleSelect}
-          onOpenChange={handleToggle}
-          selected={selected}
-          toggle={toggleCompliance}
-          shouldFocusFirstItemOnOpen={false}
-        >
-          {complianceOptions()}
-        </Select>
-      )}
+          {isSuccess && selectOptions.length === 0 && (
+            <SelectOption isDisabled>
+              {`No results found for "${filterValue}"`}
+            </SelectOption>
+          )}
+        </SelectList>
+      </Select>
+      )
```

Comment:
![image](https://github.com/user-attachments/assets/f38433c3-7608-4949-a175-25a53ab4db92)

The closing bracket will need to be removed so it doesn't render after the select.

Reply:
I wondered what that was!